scripts/convert_hf_checkpoint.py

```diff
@@ -179,6 +180,10 @@ def copy_weights_hf_llama(
             param = saver.store_early(param)
         state_dict[to_name] = param
 
+    if tie_weights:
```

Comment:
I think I prefer the previous version where we did this automatically if the key wasn't there

Reply:
Nice, I can actually reuse that for the OLMo port :)